docs/sources/developers/plugins/publish-a-plugin.md

```diff
@@ -151,11 +50,11 @@ To speed up the time it takes to review your plugin:
      - Select **Multiple** if you'd like to submit separate plugin archives for each architecture.
        This can lead to faster downloads since users can select the specific architecture on which they want to install the plugin.
    - **URL:** A URL that points to a ZIP archive of your packaged plugin.
-   - **Source Code URL:** A URL that points to a public git repository or ZIP archive of your complete plugin source code.
+   - **Source Code URL:** A URL that points to a public Git repository or ZIP archive of your complete plugin source code.
```

Comment:
I think we should provide some examples here. Seen many times when reviewing plugins that the provided source code URL not necessarily is connected to the git tag the plugin was built from. @academo you maybe can provide some more context/input here?  

This would also be applicable to update a plugin below.

Reply:
I remember when we added this part to the documentation we thought it'd be too much to put there but I leave you to decide.

Valid URls examples:


Using a tag or branch: https://github.com/grafana/clock-panel/tree/v2.1.3
A tag or branch and the code is in a subdirectory (important for mono repos): https://github.com/grafana/clock-panel/tree/v2.1.3/plugin/ (here `plugin` contains the plugin code)
Using the latest main or master branch commit: https://github.com/grafana/clock-panel/ (not recommended, better pass a tag or branch)

Reply:
Thanks. So maybe to limit the amount of details here add a question under `Frequently asked questions` like "What Source Code URL formats are supported" or something?

Reply:
I have added these examples as one of the FAQs and added a link from the procedural documentation.